src/auditor/auditor.py

```diff
@@ -152,24 +170,23 @@ def __init__(self, verbose=False):
             #: Get the groups
             if self.verbose:
                 print('Getting groups...')
-            groups = self.gis.groups.search('title:*')
+            groups = self.gis.groups.search('title:*')  # pylint: disable=no-member
             self.groups_dict = {g.title: g.id for g in groups}
 
         except HTTPError:
             print(f'Connection error, probably for connection with {credentials.ORG}')
             raise
 
-
     def read_metatable(self, table, fields):
-        '''
+        """
         Read metatable 'table' into self.metatable_dict.
 
         Returns: list of any duplicate AGOL item ids found
-        '''
+        """
 
         duplicate_keys = []
 
-        with arcpy.da.SearchCursor(table, fields) as meta_cursor:
+        with arcpy.da.SearchCursor(table, fields) as meta_cursor:  # pylint: disable=no-member
```

Comment:
**suggestion**: this disable can go away now if you bring in the updates from agrc/python
```suggestion
        with arcpy.da.SearchCursor(table, fields) as meta_cursor:
```